moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -181,9 +204,9 @@ void RobotModelLoader::loadKinematicsSolvers(const kinematics_plugin_loader::Kin
     const std::vector<std::string>& groups = kinematics_loader_->getKnownGroups();
     std::stringstream ss;
     std::copy(groups.begin(), groups.end(), std::ostream_iterator<std::string>(ss, " "));
-    ROS_DEBUG_STREAM("Loaded information about the following groups: '" << ss.str() << "'");
+    RCLCPP_DEBUG(LOGGER_ROBOT_MODEL_LOADER, "Loaded information about the following groups: '%S' ", ss.str().c_str());
```

Comment:
Please change to '%s', '%S' is not a standard specifier. (https://en.cppreference.com/w/cpp/io/c/fprintf)